tests/acceptance/Payment/MerchantAccountPaymentAuthorizationTest.php

```diff
@@ -156,6 +157,24 @@
     \expect($fetched->getId())->toBeString();
 });
 
+\it('creates payment with user political exposure', function () {
+    $helper = \paymentHelper();
+
+    $payment = $helper->client()->payment()
+        ->paymentMethod($helper->bankTransferMethod($helper->sortCodeBeneficiary()))
+        ->amountInMinor(10)
+        ->currency('GBP')
+        ->user($helper->user()->politicalExposure(UserPoliticalExposures::CURRENT))
+        ->create();
+
+    $fetched = $payment->getDetails();
```

Comment:
We probably don't need to retrieve the payment here? We could just assert an id exists on $payment from the create response.